src/atoms/walletLabel/walletLabel.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { shortenAddressTo11Chars } from '../../utils/shortenAddress';
+import { ShortenText } from '../shortenText/shortenText';
+import { Text } from '../texts/text';
+import { Container } from './styles';
+import { WalletLabelProps } from './types';
+
+export const WalletLabel = (
+  {
+    label, address, size, color, charactersToDisplay, width,
+  }: WalletLabelProps,
+) => {
+  const labelHasSpaces = label?.includes(' ') ?? false;
```

Comment:
i think you can just do !!label.includes(' ')

+ 

maybe you should for safety trim() the end of the string in case for whatever reason that was the only space